src/database/queries.ts

```diff
@@ -145,3 +140,56 @@ export const getVouchersByVendorUuid = async (
     throw e;
   }
 };
+
+/**
+ * Create a new "vendor" document and at it to the vendors collection.
+ */
+export const createVendor = async (vendor: VendorCreate): Promise<uuid> => {
+  try {
+    const docRef = await addDoc(vendorCollection, vendor);
+    await updateDoc(docRef, { uuid: docRef.id});
+    return docRef.id; 
+  } catch (e) {
+    console.warn('(createVendor)', e);
+    throw e;
+  }
+}
+
+/**
+ * Update an existing "vendor" name. 
+ */
+export const updateVendorName = async (name: string, vendor: Partial<Vendor>) => {
```

Comment:
We can create a single helper function to update vendor information per @wangannie 's comment. Feel free to look at [Sauhard's PR](https://github.com/calblueprint/vouchers4veggies/pull/10/files) in `queries.ts` line 108 to see how he did this, and it should be very similar.